lib/CipherInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Aol\Transformers;
+
+interface CipherInterface
+{
+	/**
+	 * Encrypt the given text.
+	 *
+	 * @param string $plain_text The text to encrypt.
+	 *
+	 * @return string The encrypted text.
+	 */
+	function encrypt($plain_text);
```

Comment:
Was wondering if it made sense to support encryption/decryption of non-strings as well. For instance could encrypt entire arrays of data, or other types, but not sure how far down that hole to go.